forms/collections/elements.js

```diff
@@ -12,6 +12,7 @@ define(function (require) {
   var formsErrors = require('forms/error-helpers');
 
   // this module
+  var ElementsCollection;
```

Comment:
Is there a reason why this was changed to a variable assignment from a simpler `return`? There don't seem to be any references to it within the module. Intermediate variables are fine when they make code clearer, I guess I'm just wondering how much this particular change helps.